rfm_dashboard/controller/landing_page.py

```diff
@@ -0,0 +1,9 @@
+from flask import Blueprint, render_template
+from ..models.get_rfm_data import get_rfm_data
```

Comment:
Why import the rfm_data function like `from ..models.get_rfm_data import rfm_data` na krub?

I prefer to import dependencies in the format `from <application_name>.<directory_path>.<file_name> import *` or function name, in this case should be `rfm_dashboard.models.get_rfm_data.py`